ui/app/AppLayouts/Onboarding/controls/LoginUserSelector.qml

```diff
@@ -59,7 +60,7 @@ Control {
                     label: currentEntry.item.username
                     image: currentEntry.item.thumbnailImage
                     colorId: currentEntry.item.colorId
-                    keycardCreatedAccount: currentEntry.item.keycardCreatedAccount
+                    keycardCreatedAccount: currentEntry.item.keycardCreatedAccount && root.isKeycardEnabled
```

Comment:
I don't think this is correct. If an account was created using Keycard and Keycard is not enabled, we actually need to show a warning or something.

Basically, the account will not be usable, because the keys are inside the Keycard.

So maybe instead we should disable the entry and add a tooltip (that shows on click because of mobile)

Reply:
But on a second thought, how could this happen? We never supported creating Keycard accounts on mobile before, so this is not a problem at all? Such a keycard entry would never appear then in the profile selector

Reply:
I see... so either filtering such accounts off (removing them from the selector, which I just did locally), or just making it disabled (but the LoginSelector was never designed or developed with the possibility of having some disabled items in it...)

Reply:
Fixed by filtering out the Keycard profiles if we have any and displaying this little warning icon with a tooltip:

<img width="2950" height="1988" alt="image" src="https://github.com/user-attachments/assets/9f80dba6-0a64-4c44-9728-5fff9d537ee9" />